src/py42/sdk/queries/fileevents/file_event_query.py

```diff
@@ -34,22 +34,15 @@ def __str__(self):
         groups_string = u",".join(
             str(group_item) for group_item in self._filter_group_list
         )
-
         if self.page_token is not None:
             paging_prop = u'"pgToken":"{}"'.format(self.page_token)
+
         else:
-            paging_prop = u'"pgNum":{}'.format(self.page_number)
-
-        json = (
-            u'{{"groupClause":"{0}", "groups":[{1}], "srtDir":"{2}", '
-            u'"srtKey":"{3}", {5}, "pgSize":{4}}}'.format(
-                self._group_clause,
-                groups_string,
-                self.sort_direction,
-                self.sort_key,
-                self.page_size,
-                paging_prop,
+            paging_prop = u'"srtDir":"{}", "srtKey":"{}", "pgNum":{}'.format(
```

Comment:
Diff looks weird, but this makes it so that `srtKey` and `srtDir` are added in the `else` condition, so only if not using `pgToken`